crowbar_framework/app/models/proposal_queue.rb

```diff
@@ -0,0 +1,35 @@
+class ProposalQueue < ActiveRecord::Base
+  # FIXME: remove this when the export is properly implemented
+  class_attribute :chef_type
+
+  self.chef_type = "data_bag_item"
+  # only accept one default record named "queue"
+  self.primary_key = "name"
+
+  serialize :properties, Utils::JSONWithIndifferentAccess
+
+  def proposals
+    ProposalQueue.find("queue").properties["proposal_queue"]
+  end
+
+  def <<(item)
+    pq = ProposalQueue.find("queue")
```

Comment:
@aspiers should have tested it before. that actually doesn't work...
`NoMethodError: undefined method `find' for #<ProposalQueue:0x0000000559b7f8>`